windows/installer-source/KaliteSetupScript.iss

```diff
@@ -437,9 +441,19 @@ begin
   
     ShellExec('open', 'taskkill.exe', '/F /T /im "KA Lite.exe"', '', SW_HIDE, ewWaitUntilTerminated, killErrorCode)
     ShellExec('open', 'tskill.exe', ' "KA Lite"', '', SW_HIDE, ewWaitUntilTerminated, killErrorCode);
-
     RegDeleteValue(HKCU, 'SOFTWARE\Microsoft\Windows\CurrentVersion\Run', ExpandConstant('{#MyAppName}'));
-   
+    if(MsgBox('Warning! All python process will be closed in order to install KA Lite.' + #13#10 + #13#10 + 'Click OK to continue.', mbConfirmation, MB_OKCANCEL) = idCancel) then

```

Comment:
@radinamatic I updated the confirmation message as you suggested. Here's the [windows installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMY25aV0NVUDRWV2c) to test it